app/views/courses/CoursesNotAssignedModal.coffee

```diff
@@ -0,0 +1,9 @@
+ModalView = require 'views/core/ModalView'
+template = require 'templates/courses/courses-not-assigned-modal'
+
+module.exports = class CoursesNotAssignedModal extends ModalView
+  id: 'courses-not-assigned-modal'
+  template: template
+
+  initialize: (options) ->
+    _.assign(@, _.pick(options, 'selected', 'totalSpotsAvailable', 'unenrolledStudents'))
```

Comment:
Yeah, something to consider in September. A lot of component libraries that I see set the context to a model or data object (Marionette, Aurelia, Vue) rather than the view directly. Something to consider in September :)